### PR TITLE
Fixed binding of YUV.

### PR DESCRIPTION
**Pull request by emilwestergren**

### CODE REVIEW

File: source/draw/gpu/GpuSurface.ooc [hunk 1]
```diff
@@ -83,7 +83,13 @@ GpuSurface: abstract class extends Canvas {
...
 		else
 			Debug raise("Invalid image type in GpuSurface!")
-		map add("texture0", temporary)
+		if (temporary instanceOf?(GpuYuv420Semiplanar)) {
+			yuv := temporary as GpuYuv420Semiplanar
+			map add("texture0", yuv y)
+			map add("texture1", yuv uv)
+		}
+		else
+			map add("texture0", temporary)
```
Comment (niklasborwell) [inline source/draw/gpu/GpuSurface.ooc]: } else

Comment (davidhesselbom) [inline source/draw/gpu/GpuSurface.ooc]: @thomasfanell This one should be a piece of cake for magic.

Comment (davidhesselbom) [inline source/draw/gpu/GpuSurface.ooc]: Well, if it weren't for version blocks...

Comment (emilwestergren) [inline source/draw/gpu/GpuSurface.ooc]: Oh yes, the ugly `else` format rule, how could I forget... We probably want it in magic, I will never get used to it.

**Comment by niklasborwell** [comment]
Is there a reason why the appveyor test has started failing again?

**Comment by davidhesselbom** [comment]
Duno, but it seems to always have to do with `Promise`. Maybe something in the Windows implementation broke when the threading classes in the SDK were cleaned up, I don't know.

**Pull request merged**